src/bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt

```diff
@@ -98,6 +98,20 @@
         </div>
       </tal:lab_analyses>
 
+      <tal:ast_analyses>
+        <div class="row">
+          <div class="col-sm-12">
+            <h3>
+              <i class="labanalyses-icon" title="AST" i18n:attributes="title">
+                <svg tal:replace="structure senaite_theme/icon_data/analysisservice" />
+              </i>
+              <span i18n:translate="">Antibiotic Susceptibility</span>
+            </h3>
+            <span tal:replace="structure python:view.render_analyses_table(table='ast')"/>
+          </div>
+        </div>
+      </tal:ast_analyses>
```

Comment:
This probably does not belong to the PR

Reply:
Yes apologies... this is the good one: https://github.com/senaite/senaite.core/pull/1687